assets/js/components/BatchEdit/About/CoreMetadata.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import UITagNotYetSupported from "../../UI/TagNotYetSupported";
+import UIFormBatchEDTFDate from "../../UI/Form/BatchEDTFDate";
```

Comment:
Anything new we add, let's make sure we're always using the relative import pattern to save us time down the road:   `@js/components/UI/Form/BatchEDTFDate`